example/src/screens/Room.tsx

```diff
@@ -32,20 +44,130 @@ export const Room = ({ navigation }: Props) => {
   const participants = Membrane.useRoomParticipants();
   const [focusedParticipantData, setFocusedParticipantData] =
     useState<Participant | null>(null);
+  // This string array will containg strings in a form of participantId+trackId
+  // to differentiate between camera and screenshare video tracks.
+  const [participantsOrder, setParticipantsOrder] = useState<string[] | null>(
```

Comment:
are you sure this should be in state and not in ref? setting state when rendering sounds bad to me

Reply:
when vad status changes, the room will re-render anyway, because `participants` array (the one we got from `useRoomParticipants` hook) changes, right? so I think we're making unecessary re-render here:
1) order is kept in the state
- participants are initially rendered in the old order
- in render the new order is set
- room is rendered again with the new order
2) order is kept in the ref
- the new order (participants array) is calculated based on current vad status and data in `participantsOrder` and the participants are rendered in the new order on the first time, no additional re-render is needed

at least that's how I understand your code, it's complicated, let me know what do you think 😅

Reply:
Yeah, prob. it can be done with useRef, if we want to spare that one render